### PR TITLE
Phase 1 by branch Register

### PR DESCRIPTION
**Pull request by upasna-talworx**
## What does this PR do?

- Fixes #XXXX (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)

## Mandatory Tasks (DO NOT REMOVE)

- [ ] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [ ] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [ ] I confirm automated tests are in place that prove my fix is effective or that my feature works.

## How should this be tested?

- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

File: apps/web/components/getting-started/steps-views/UserProfile.tsx [hunk 1]
```diff
@@ -11,16 +13,38 @@ import { trpc } from "@calcom/trpc/react";
 import { Button, Editor, ImageUploader, Label, showToast } from "@calcom/ui";
 import { UserAvatar } from "@calcom/ui";
 
-type FormData = {
-  bio: string;
+import { ZAdminSetupInputSchema } from "/home/upasnasingh/talworx/skillgram/packages/trpc/server/routers/loggedInViewer/setupAdmin.schema";
```
Comment (AnshulTalworx) [inline apps/web/components/getting-started/steps-views/UserProfile.tsx]: Please use relative paths. Absolute paths will not work on the server. Use @calcom/trpc...

**Review by AnshulTalworx — COMMENTED** [review]
There is only one comment rest is fine.

File: apps/web/components/getting-started/steps-views/UserProfile.tsx (hunk 1 above)
Comment (upasna-talworx) [reply apps/web/components/getting-started/steps-views/UserProfile.tsx]: Sure, Anshul! That was accidentally overlooked, will fix this.

Comment (AnshulTalworx) [reply apps/web/components/getting-started/steps-views/UserProfile.tsx]: Also, please remove the Privacy Policy and Terms notice from the Signup page.

File: apps/web/modules/signup-view.tsx [hunk 2]
```diff
@@ -624,88 +619,6 @@ export default function Signup({
               </div>
```
Comment (AnshulTalworx) [inline apps/web/modules/signup-view.tsx:605]: Remove the Signup terms

**Pull request merged**